datacite.yml

```
@@ -6,6 +6,11 @@ authors:
     lastname: Hanke
     id: "ORCID:0000-0001-6398-6370"
 
+  - firstname: Adina
+    lastname: Wagner
+    id: "ORCID:0000-0003-2917-3450"
+
+
```

```suggestion

  - firstname: Laura
    lastname: Waite
    id: "ORCID:0000-0003-2213-7465"
```

@loj should be here, too, correct @christian-monch ?